src/components/common/Input.tsx

```diff
@@ -0,0 +1,38 @@
+type Props = {
+  label: string;
+  name: string;
+  id: string;
+  placeholder?: string;
+  value: string;
+  OnChange: (value: string) => void;
```

Comment:
```suggestion
  onChange: (value: string) => void;
```
Just a small suggestion in here 😁